### PR TITLE
Empty message to ServiceBusTrigger breaks the extension. Fixes #2573, #1644

### PR DESCRIPTION
**Pull request by alrod**

### CODE REVIEW

**Comment by alrod** [comment]
I added a test in updated PR

File: src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs [hunk 1]
```diff
@@ -73,9 +73,10 @@ public string ToInvokeString()
             return Convert.ToBase64String(clonedMessage.Body);
         }
 
-        private static Task<string> GetBytesLengthAsync(Message clonedMessage)
+        private static Task<string> GetDescriptionAsync(Message clonedMessage)
         {
-            string description = string.Format(CultureInfo.InvariantCulture, "byte[{0}]", clonedMessage.Body.Length);
+            string description = clonedMessage.Body != null
```
Comment (pragnagopa) [inline src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs]: Add null checks in : GetBase64StringAsync and GetTextAsync

File: test/Microsoft.Azure.WebJobs.ServiceBus.UnitTests/MessageToStringConverterTests.cs [hunk 2]
```diff
@@ -50,9 +50,11 @@ public class MessageToStringConverterTests
         [InlineData(ContentTypes.ApplicationOctetStream, TestString)]
         [InlineData(null, TestJson)]
         [InlineData("application/xml", TestJson)]
+        [InlineData(ContentTypes.TextPlain, null)]
+        [InlineData(ContentTypes.TextPlain, "")]
         public async Task ConvertAsync_ReturnsExpectedResult_WithSerializedString(string contentType, string value)
         {
-            Message message = new Message(Encoding.UTF8.GetBytes(value));
+            Message message = new Message(value == null ? null : Encoding.UTF8.GetBytes(value));
```
Comment (pragnagopa) [inline test/Microsoft.Azure.WebJobs.ServiceBus.UnitTests/MessageToStringConverterTests.cs]: Add similar tests for null and empty in ConvertAsync_ReturnsExpectedResult_WithBinarySerializer test

File: src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageToStringConverter.cs [hunk 3]
```diff
@@ -20,6 +20,10 @@ internal class MessageToStringConverter : IAsyncConverter<Message, string>
             {
                 throw new ArgumentNullException("input");
             }
+            if (input.Body == null)
```
Comment (mathewc) [inline src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageToStringConverter.cs]: What happens end to end when null is returned here? Does the user function get invoked with a null or empty string?

File: src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs (hunk 1 above)
Comment (alrod) [reply src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageValueProvider.cs]: Added

File: test/Microsoft.Azure.WebJobs.ServiceBus.UnitTests/MessageToStringConverterTests.cs (hunk 2 above)
Comment (alrod) [reply test/Microsoft.Azure.WebJobs.ServiceBus.UnitTests/MessageToStringConverterTests.cs]: Added only null, DataContractBinarySerializer converts empty string to @string3http://schemas.microsoft.com/2003/10/Serialization/
